### PR TITLE
Commands SpawnCar

### PR DESCRIPTION
**Pull request by Barwoz**

### CODE REVIEW

File: src/core/events/client/car.lua [hunk 1]
```diff
@@ -0,0 +1,9 @@
+NCS:registerNetEvent("ncs_core:setVehicleProperties")
+NCS:handleEvent("ncs_core:setVehicleProperties", function(model, playerCoords, playerHeading, props)
+    local vehicle = API_Vehicles:spawn(model, playerCoords, playerHeading, true)
+    while not DoesEntityExist(vehicle) do
+        Wait(0)
+    end
+    API_Ped:setIntoVehicle(GetPlayerPed(-1), vehicle, -1)
```
Comment (CsokiHUN) [inline src/core/events/client/car.lua:7]:
```suggestion
     API_Ped:setIntoVehicle(PlayerPedId(), vehicle, -1)
```

**Review by CsokiHUN — CHANGES_REQUESTED** [review]

File: src/core/events/client/car.lua (hunk 1 above)
Comment (LQuatre) [reply src/core/events/client/car.lua:7]: it's same result

**Review by MoskalykA — CHANGES_REQUESTED** [review]
Syntax to be reviewed (please see the core syntax in general) and the logic (a server side function exists to display a vehicle: https://github.com/NextCitizens/ncs_core/blob/main/src/api/vehicles/server/spawnVehicle.lua)

**Comment by Barwoz** [comment]
> Syntax to be reviewed (please see the core syntax in general) and the logic (a server side function exists to display a vehicle: https://github.com/NextCitizens/ncs_core/blob/main/src/api/vehicles/server/spawnVehicle.lua)

Yes I have used this server side function. For syntax yes I am sorry, I will check it out and edit.

File: src/localization/locales/lang_hu.lua [hunk 2]
```diff
@@ -33,4 +33,8 @@ _Literals = {
     KICK_DEFAULT_MESSAGE = "Ki lettél dobva a szerverről.",
     BAN_DEFAULT_MESSAGE = "Ki lettél tiltva a szerverről.",
     UNBAN_DEFAULT_MESSAGE = "A kitiltásod feloldva.",
+
+    ERROR_CAR_NO_MODEL = "Nem található a jármű modellje.",
+    SUCCESS_CAR_SPAWNED = "A jármű elindult.",
```
Comment (CsokiHUN) [inline src/localization/locales/lang_hu.lua:38]:
```suggestion
    SUCCESS_CAR_SPAWNED = "Sikeres jármű lehívás.",
```

**Review by CsokiHUN — CHANGES_REQUESTED** [review]

**Comment by LQuatre** [comment]
> Syntax to be reviewed (please see the core syntax in general) and the logic (a server side function exists to display a vehicle: https://github.com/NextCitizens/ncs_core/blob/main/src/api/vehicles/server/spawnVehicle.lua)

ServerSide functions is broken ://

**Comment by MoskalykA** [comment]
Waiting for the redesign of the server-side vehicle system

**Pull request closed without merging**